File: src/su/test/Wrttracs/mklines.f
      	integer i, j, ncurves, npoints, stdout, cf
      	parameter (ncurves = 2, npoints = 6, stdout = 6, cf = 8)
	real x(1:npoints), y(1:npoints)

*     	...Open output curve file.
 10  	open(unit = cf, file = 'LINES', access = 'sequential',
     :	     form = 'unformatted', status = 'new', err = 20)
      	go to 30
 20     write(stdout, '(a)') ' ***Can not open curve file.'
        stop

 30	do 50 j = 1, ncurves
         	write(cf) npoints
      		do 40 i = 1, npoints
			x(i) = float(10*(j-1) + i-1)
			y(i) = float(i-1)
 40   		continue
         	write(cf) (x(i), y(i), i = 1, npoints) 
 50	continue

      	stop
      	end

File: src/su/test/Wrttracs/mklines.c
/* Make some test traces */

#define	STDOUT	1
main()
{
	int i, j, npoints=6, ncurves=2;
	float x, y;

	for (j = 0; j < ncurves; ++j) {
		write(STDOUT, &npoints, sizeof(int));
		for (i = 0; i < npoints; ++i) {
			x = (float) 10*j + (float) i;
			y = (float) i;
			write(STDOUT, &x, sizeof(float));
			write(STDOUT, &y, sizeof(float));
		}
	}
}
